nautobot/project-static/css/dark.css

```diff
@@ -32,12 +32,11 @@ html[data-theme="dark"] .color-block {
 /* Dropdown list when selecting a color for status, device role, etc */
 html[data-theme="dark"] .select2-results__option {
   filter: invert(1) hue-rotate(180deg);
-  color: #ccc !important;
+  color: #ccc;
 }
 /* Dropdown list for showing selected color for status, device role, etc */
 html[data-theme="dark"] .select2-selection__rendered {
   filter: invert(1) hue-rotate(180deg);
-  color: #ccc !important;
 }
```

Comment:
<img width="905" alt="Screen Shot 2022-09-12 at 4 03 48 PM" src="https://user-images.githubusercontent.com/46973263/189746269-f492bb31-bf41-4083-b486-3612f23f7f86.png">
I have tried that but the colors in the dropdown are rendering incorrectly. I think since select2-results__option and select2-selection__rendered already inverted the color and inverting it again will turn them into the wrong colors. IMHO, the best solution is to change the font color to something obvious and readable for both dropdowns with plain backgrounds and dropdowns with colored backgrounds. What do you think? @bryanculver

Reply:
Delete `!important` from `color: #ccc;` to prevent this attribute from overriding the font color in the color selection dropdowns which makes the text very hard to read.

Reply:
Per my proposal it renders this:

<img width="579" alt="image" src="https://user-images.githubusercontent.com/31187/189901578-68f97c35-849f-4a72-a314-47ff5a1d52c8.png">

And all other drop downs render as: 
<img width="386" alt="image" src="https://user-images.githubusercontent.com/31187/189901679-229b70bd-7a18-4ad5-89a1-cd759e504ccb.png">

Reply:
I see you mean your request by itself. I put it on top of what we already have.
Also, did you try selecting one of the choices in the second dropdown? On my end it rendered like this, is this acceptable?
<img width="391" alt="Screen Shot 2022-09-13 at 9 22 57 AM" src="https://user-images.githubusercontent.com/46973263/189912419-acf313ff-902e-4db6-bfc2-8648036563b8.png">

Reply:
Nevermind it is fixed by ignoring inverting on the `select2-selection__choice` class. All good now
<img width="393" alt="Screen Shot 2022-09-13 at 9 30 27 AM" src="https://user-images.githubusercontent.com/46973263/189914229-323bd85d-5014-4b97-84c1-8d215a256319.png">